dev/webpack/config/dev.config.js

```diff
@@ -0,0 +1,17 @@
+const path = require("path");
+const webpack = require("webpack");
+const webpackBase = require("./base.config");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = webpackBase({
+    plugins: [
+        new webpack.HotModuleReplacementPlugin(),
+        new MiniCssExtractPlugin({
```

Comment:
Created #5 to consolidate the MiniCssExtractPlugin config in the `base.config.js`.